docs/modules/reference/pages/provisioning/detectors/JdbcDetector.adoc

```diff
@@ -0,0 +1,74 @@
+
+= JDBCDetector
+
+The JDBCDetector connects to a JDBC-enabled database and tries to fetch its catalog.
+If the fetch is successful, it returns a `detected` value.
+If it is not successful, it returns a `not detected` value.
+
+== Detector facts
+
+[options="autowidth"]
+|===
+| Implementation | `org.opennms.netmgt.provision.detector.jdbc`
+|===
+
+== Configuration and use
+
+.Parameters for the JDBC detector.
+[options="header, autowidth"]
+[cols="1,4,1"]
+|===
+| Parameter
+| Description
+| Default value
+
+3+| *Required*
+
+| dbDriver
+| Database driver class name.
+| org.postgresql.Driver
+
+| password
+| JDBC password.
+| Empty String object.
+
+| url
+| JDBC URL to query.
+Note: this URL can differ based on JDBC drivers, databases, and so on.
+| `"jdbc:postgresql://" + OPENNMS_JDBC_HOSTNAME + "/opennms"`
+
+| user
+| JDBC username.
+| postgres
+
+3+| *Optional*
+
+| ipMatch
+| IP address to match.
+|
+
+| port
+| Port to query.
+|
+
+| retries
+| Number of times detector should try to connect to database.
+|
+
+| serviceName
+| Description
+|
+
```

Comment:
`serviceName` is not part of the JDBC config
```suggestion
```

Reply:
I'm assuming the config parameters have been updated since 2021? The screenshot attached to the ticket includes this and the ipMatch parameter.

Reply:
That might be the case. I didn't check against the current version of Horizon.

Reply:
while we are on the topic, retries and timeout are also standard detector parameters.

Reply:
@emarsh1995  Let's go the route of what we did with the Service Monitors. The intro file has a "common configuration parameters" section that lists those specific to monitors. We then cross reference to them in each monitor, with a "This monitor implements the Common Configuration Parameters". 

This expands the scope of this ticket and the time it will take, but that's ok.